docs/background/make_models.rst

```diff
@@ -0,0 +1,121 @@
+.. _background_make_models:
+
+Make background models
+======================
+
+Gammapy tools to produce background models.
+
+Make cube background models
+---------------------------
+
+The ``gammapy-make-bg-cube-models`` command line tool can be used to produce
+background cube models from the data files at a given path location.
+
+For more details, please refer to `~gammapy.scripts.make_bg_cube_models`.
+
+Examples
+~~~~~~~~
+
+The ``gammapy-make-bg-cube-models`` tool has a few options. For a full list of options, please use:
+
+.. code-block:: bash
+
+    $ gammapy-make-bg-cube-models --help
+
+at the command line.
+
+Command examples:
+
+* Create background models using a H.E.S.S. or H.E.S.S.-like dataset
+  (can take a few minutes):
+
+  .. code-block:: bash
+
+      $ gammapy-make-bg-cube-models /path/to/fits/event_lists/base/dir \
+                                    HESS bg_cube_models
+
+* Run a quick test using only a few runs:
+
+  .. code-block:: bash
+
+      $ gammapy-make-bg-cube-models /path/to/fits/event_lists/base/dir \
+                                    HESS bg_cube_models --test
+
+* Create background models using the method developped by Michael
+  Mayer (almost equal to the default case for now):
```

Comment:
@cdeil should I add a link to Michi's thesis as a reference? I think it would be nice. Plus I don't feel like adding a long description about his method in the docs.